[DOCS] Lens Visualization Plugin Available for Cases (#1018)
elastic/security-docs: docs/cases/cases-overview.asciidoc

* Refresh the case to retrieve the latest updates

NOTE: Comments can also contain Markdown syntax and Timeline links.

[float]
[[cases-lens-visualization]]
== Add a Lens visualization

beta[]

Add a Lens visualization to your case to portray event and alert data through charts and graphs.

> IMPORTANT: Set an absolute time range for your visualization. This ensures your visualization doesn't change over time after you save it to your case and provides important context for other viewers.

Reviewer: This admonition seems more helpful to include in the procedure below, in the step where the user is actually creating/configuring the visualization.

[role="screenshot"]
image::images/add-vis-to-case.gif[Shows how to add a visualization to a case]

> To add a Lens visualization to your case within a comment:

Reviewer: This could be misread to say that the case is within a comment. Maybe something like this? Or add a step at the beginning of the procedure to create/open the comment?
```suggestion
To add a Lens visualization to a comment within your case:
```

> . Click the *Visualization* button. The Add Visualization modal appears.  

Reviewer: To match the sentence-case label as seen in the GIF. Also, "modal" seems like dev jargon to me, vs. "dialog" or "dialog box," either of which appear to be more common in other Elastic docs.
```suggestion
. Click the *Visualization* button. The Add visualization dialog appears.  
```

[float]
[[cases-lens-visualization]]
== Add a Lens visualization

beta[]

Add a Lens visualization to your case to portray event and alert data through charts and graphs.

[role="screenshot"]
image::images/add-vis-to-case.gif[Shows how to add a visualization to a case]

To add a Lens visualization to a comment within your case:

. Click the *Visualization* button. The Add visualization dialog appears.  
. Select an existing visualization from your Visualize Library or create a new visualization.

+

IMPORTANT: Set an absolute time range for your visualization. This ensures your visualization doesn't change over time after you save it to your case and provides important context for other viewers.

+
. Save the visualization to your Visualize Library by clicking the *Save to library* button (optional).
.. Enter a title and description for the visualization.  
.. Choose whether you want to keep the *Update panel on Security* activated. This option is activated by default and automatically adds the visualization to your Visualize Library.
. After you've finished creating your visualization, click *Save and return* to go back to your case.

> . Click *Preview* to check how the visualization appears in the case comment.

Reviewer 2: ```suggestion
. Click *Preview* to see how the visualization will appear in the case comment.
```

. Click *Add Comment* to add the visualization to your case.  

> NOTE: Once a visualization has been added to a case, it cannot be modified or deleted. However, you can further interact with the visualization by clicking the *Open Visualization* option in the comment menu.

Reviewer 2: ```suggestion
NOTE: Once a visualization has been added to a case, it cannot be modified or deleted. However, you can interact with the visualization by clicking the *Open Visualization* option in the comment menu.
```
